packages/pages/src/common/src/template/internal/getTemplatesFilepath.test.ts

```diff
@@ -0,0 +1,49 @@
+import path from "path";
+import { Path } from "../../project/structure.js";
+import { getTemplatesFilepath } from "./getTemplatesFilepath.js";
+import minimatch from "minimatch";
+
+const rootPath = "src/templates";
+const domain1Path = "src/templates/some.domain1.com";
+const domain2Path = "src/templates/some.domain2.com";
+jest.mock("glob", () => {
+  return {
+    sync: (glob: string) => {
+      const filepaths = [
+        `${domain1Path}/brand1.tsx`,
+        `${domain1Path}/test.tsx`,
+        `${domain2Path}/brand2.tsx`,
+        `${domain2Path}/test.tsx`,
+        `${rootPath}/share.tsx`,
+        `${rootPath}/test.tsx`,
+      ];
+      return filepaths.filter((f) => minimatch(path.resolve(f), glob));
+    },
+  };
+});
+afterAll(() => jest.unmock("glob"));
```

Comment:
Is this unmock necessary so it doesn't bleed into other tests? I thought that jest.mock would only last for the current test file but could be wrong

Reply:
This was copied from `loader.test.ts`. I thought you would need `resetModules` / `resetMocks` in the jest config to have it auto unmock but it looks like it's fine without it! I cloned this test fie, removed the mock glob code, and it failed so this mock didn't bleed into other test. I will remove